### PR TITLE
Add a notebook for llama2 serving and finetuning with peft, and analyze with vertex text moderation API

### PR DESCRIPTION
**Pull request by genquan9**
**REQUIRED:** Add a notebook for llama2 serving and finetuning with peft, and analyze with vertex text moderation API.

<br>
Add a notebook for llama2 serving and finetuning with peft, and analyze with vertex text moderation API
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [Y] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [Y] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb [hunk 1]
```diff
@@ -0,0 +1,756 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>I think the NOTE is unneccessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>Add some explanation what the functions do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>serving in Vertex -&gt; serving in Vertex AI</p><p> </p><p>gcs -&gt; Cloud Storage</p><p> </p><p>huggingface -&gt; Hugging Face</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>We -&gt; You</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>add option to delete the bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (genquan9) [reply notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>sg. removed. </p>

Comment (genquan9) [reply notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>done</p>

[the remark this answers is not in the document]
Comment (genquan9) [reply notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>DONE</p>

Comment (genquan9) [reply notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb:1]: <p>done</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**